content/en/blog/_posts/Cluster-API-v1alpha3-Delivers-New-Features-and-an-Improved-User-Experience.md

```diff
@@ -0,0 +1,209 @@
+
+> #### What is a failure domain? [side bar]
+> A failure domain is a way to group the resources that would be made unavailable by some failure. For example, in many public clouds, an “availability zone” is the default failure domain. A zone corresponds to a data center. So, if a specific data center is brought down by a power outage or natural disaster, all resources in that zone become unavailable. If you run Kubernetes on your own hardware, your failure domain might be a rack, a network switch, or power distribution unit.
+
+The Kubeadm-based ControlPlane distributes nodes across failure domains. To minimize the chance of losing multiple nodes in the event of a domain outage, it tries to distribute them evenly: it deploys a new node in the failure domain with the fewest existing nodes, and it removes an existing node in the failure domain with the most existing nodes.
+
+MachineDeployments and MachineSets do not distribute nodes across failure domains. To deploy your worker nodes across multiple failure domains, create a MachineDeployment or MachineSet for each failure domain.
+
+The Failure Domain API works on any infrastructure. That’s because every Infrastructure Provider maps failure domains in its own way. The API is optional, so if your infrastructure is not complex enough to need failure domains, you do not need to support it. This example is for the Cluster API Docker Infrastructure Provider. Note that two of the domains are marked as suitable for control plane nodes, while a third is not. The Kubeadm-based ControlPlane will only deploy nodes to domains marked suitable.
+
+```yaml
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
+kind: DockerCluster
+metadata:
+  name: example
+spec:
+  controlPlaneEndpoint:
+    host: 172.17.0.4
+    port: 6443
+  failureDomains:
+    domain-one:
+      controlPlane: true
+    domain-two:
+      controlPlane: true
+    domain-three:
+      controlPlane: false
+```
+
+The [AWS Infrastructure Provider](https://github.com/kubernetes-sigs/cluster-api-provider-aws) (CAPA), maintained by the Cluster API project, maps failure domains to AWS Availability Zones. Using CAPA, you can deploy a cluster across multiple Availability Zones. First, define subnets for multiple Availability Zones. The CAPA controller will define a failure domain for each Availability Zone. Deploy the control plane with the KubeadmControlPlane: it will distribute replicas across the failure domains. Finally, create a separate MachineDeployment for each failure domain.
+
+## Automated Replacement of Unhealthy Nodes
+#### Special thanks to [Alberto García Lamela](https://github.com/enxebre), and [Joel Speed](http://github.com/joelspeed)
```

Comment:
Are these headings? The page will be more accessible if you don't mark this as a heading.

Reply:
Can you recommend formatting, please?

Reply:
You can use HTML, or italics, eg:
```markdown
_Special thanks to [Alberto García Lamela](https://github.com/enxebre), and [Joel Speed](http://github.com/joelspeed)_
```